### PR TITLE
EnergyStar version x.2 

### PR DESCRIPTION
**Pull request by bpark1327**
## Pull Request Description

Implement EnergyStar SFNH National v3.2 and MFNC National v1.2

## Checklist

Not all may apply:

- [x] ~OS-HPXML git subtree has been pulled~
- [x] 301/ES rulesets and unit tests have been updated
- [x] 301validator.xml has been updated (reference EPvalidator.xml)
- [x] Workflow tests have been updated
- [x] Documentation has been updated
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected regression test changes on CI

### CODE REVIEW

File: rulesets/EnergyStarRuleset/resources/EnergyStarRuleset.rb [hunk 1]
```diff
@@ -908,7 +909,10 @@ def self.set_appliances_clothes_washer_reference(orig_hpxml, new_hpxml)
       location = clothes_washer.location.gsub('unvented', 'vented')
     end
 
-    reference_values = HotWaterAndAppliances.get_clothes_washer_default_values(@eri_version)
+    if [ESConstants.SFNationalVer3_2].include? @program_version  # FIXME: Better way to handle it?
+      std_models = 'Std 2018-present'
+    end
+    reference_values = HotWaterAndAppliances.get_clothes_washer_default_values(@eri_version, std_models)
```
Comment (shorowit) [inline rulesets/EnergyStarRuleset/resources/EnergyStarRuleset.rb:915]: I suggest not touching OS-HPXML (revert the change) and just set the clothes washer values here in this method if it's version x.2.

File: rulesets/EnergyStarRuleset/resources/constants.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def self.SFVersions
   end
 
   def self.MFVersions
-    return [self.MFNationalVer1_0, self.MFNationalVer1_1, self.MFNationalVer1_2, self.MFOregonWashingtonVer1_2]
+    return [self.MFNationalVer1_0, self.MFNationalVer1_1, self.MFOregonWashingtonVer1_2]
```
[the remark this answers is not in the document]
Comment (bpark1327) [reply rulesets/EnergyStarRuleset/resources/constants.rb:21]: Yes, it's just a temporary change. I am going to revert it when it's ready.

**Comment by shorowit** [comment]
Waiting on:
- [ ] EPA water heater FHR value
- [ ] Fix for https://github.com/NREL/OpenStudio-HPXML/issues/1115

**Comment by shorowit** [comment]
@bpark1327 Can you pull in the latest master and resolve the conflict?

**Review by shorowit — APPROVED** [review]
Looks good to me. I wrapped up the final few small things. Great work, @bpark1327.
